.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow should be triggered when a release is done (workflow_call)
+# And also need to be able to run manually (workflow_dispatch). It uses
+# the current version from poetry as a tag for the docker container and 
+# pushes the container to ghcr.
+
+name: "Publish to GHCR"
+
+on: [workflow_call, workflow_dispatch]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: downcase REPO
```

Comment:
what does this do?

Reply:
convert Deltares/D-EcoImpact to deltares/d-ecoimpact... containers can't be registered with uppercases in the name. Also not sure if it isn't a best practise to have a lowercase github name. 

Will add a comment